catalog/transformer/catalog-transformer-pdf/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD description="PDF Input Transformer"
+         name="PDF Input Transformer"
+         id="ddf.catalog.transformer.input.pdf.PdfInputTransformer">
+
+        <AD description="Use PDF metadata title field as the metacard title attribute."
```

Comment:
Need to say what the default case is.  eg, `Use PDF metadata title field as the metacard title attribute.  By default use the filename as the title.`